contracts/test/PoolFDTLosses.t.sol

```diff
@@ -0,0 +1,314 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.11;
+pragma experimental ABIEncoderV2;
+
+import "./TestUtil.sol";
+
+contract PoolClaimTest is TestUtil {
+
+    using SafeMath for uint256;
+
+    function setUp() public {
+        setUpGlobals();
+        setUpTokens();
+        setUpOracles();
+        setUpFactories();
+        setUpCalcs();
+        setUpActors();
+        setUpBalancerPool();
+        setUpLiquidityPools();
+        createLoans();
+    }
+
+    function getSpecs(
```

Comment:
This is only used once in the `getLoanAndDepositAmount` function, so lets just move that logic in